python/doc/theory/probabilistic_modeling/conditional_distributions.rst

```diff
@@ -0,0 +1,402 @@
+is a tensorized product of the same univariate kernel.
+
+Let :math:`\inputRV` be a random vector of dimension :math:`\inputDim` defined by a  Kernel Mixture distribution based on the sample
+:math:`(\vect{s}, \dots, \vect{s}_\sampleSize)` and the kernel :math:`K`. In the continuous case, :math:`k` is the kernel PDF and we have:
+
+.. math::
+
+    p_\inputRV(\vect{x}) = \sum_{q=1}^\sampleSize \dfrac{1}{\sampleSize} p_q(\vect{x})
+
+where :math:`p_q` is the  kernel  normalized by the bandwith :math:`h`:
+
+.. math::
+
+    p_q(\vect{x}) = \prod_{j=1}^\inputDim \dfrac{1}{h^j}k\left( \dfrac{x^j- s_q^j}{h^j} \right)
+
+
+Following the Mixture case, we still have the relation :eq:`mixtureCond`. As the multivariate kernel is the tensorized product of the univariate
+kernel, we get:
+
+.. math::
+
+    \dfrac{p_q(\vect{x}_{\overline{\cI}}, \vect{x}_{\cI})}{p_{q,\cI}(\vect{x}_\cI)} = \prod_{j \in \overline{\cI}}
+    \dfrac{1}{h^j}k\left( \dfrac{x^j- s_q^j}{h^j} \right)
+
+
+Conclusion: The conditional distribution of a Kernel Mixture is a Mixture which atoms are the tensorized product of the kernel on the remaining components
+:math:`\vect{x}_\cI` and which weights :math:`\alpha_q` are proportional to:
+
+.. math::
+
+    \alpha_q \propto p_{q,\cI}(\vect{x}_\cI) = \prod_{j \in\cI} \dfrac{1}{h^j}k\left( \dfrac{x^j- s_q^j}{h^j} \right)
+
+as we have :math:`w_k = 1/N` in :eq:`mixtureCond`.
+
+
+**Truncated distributions:**  Let :math:`\inputRV` be a random vector of dimension :math:`\inputDim` which PDF is :math:`p_\inputRV`. Then,
+the truncated random vector :math:`\inputRV_T = \inputRV|\inputRV\in [\vect{a}, \vect{b}]` has the following PDF:
+
+.. math::
+
+    p_{\inputRV_T}(\vect{x}) = \dfrac{1}{\alpha} p_{\inputRV}(\vect{x}) 1_{[\vect{a}, \vect{b}]}(\vect{x})
+
+where :math:`\alpha = \Prob{\inputRV\in [\vect{a}, \vect{b}]}`.
+
+The margin of index :math:`\cI` of :math:`\inputRV_T`  has the following PDF:
+
+.. math::
+
+    p_{\inputRV_{T, \cI}}(\vect{x}) = \dfrac{1}{\alpha} \int p_{\inputRV}(\vect{x}) 1_{[\vect{a}, \vect{b}]}(\vect{x}) d\vect{x}_{\overline{\cI}}
+
+
+The conditional distribution of :math:`\inputRV_{T, \overline{\cI}}` is written as:
+
+.. math::
+
+    p_{\inputRV_{T,\overline{\cI}}|\inputRV_{T, \cI} = \vect{x}_\cI}(x_{\overline{\cI}})  =
+         \dfrac{p_{\inputRV_T}(\vect{x}_{\overline{\cI}}, \vect{x}_{\cI})}{p_{\inputRV_{T,\cI}}(\vect{x}_{\cI})}
+
+Conclusion: There is no simplification: the conditional distribution of a truncated distribution is not the truncated distribution of the conditional
+distribution.
+
+The following figure illustrates the case where :math:`(X_0, X_1) \sim \cN \left(\vect{0}, \vect{1}, \mat{R}  \right)` with :math:`R(0,1) = 0.8`.
+We plot:
+
+- the PDF of :math:`\inputRV|\inputRV\in [-0.5, 1.0]` conditioned by :math:`X_0 = 0.5` (Cond dist of truncated),
+- the truncation to :math:`[-0.5, 1.0]` of  :math:`\inputRV|X_0 = 0.5`: (Truncation of cond dist).
+
+
+.. figure:: Illustr_Trunc_cond.png
```

Comment:
did you forgot to add that file ?

Reply:
I add it, now I am sure of the result.... :-)